Postman/PostSeries.js

```diff
@@ -0,0 +1,148 @@
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+import BookModel from 'models/Book';
+import Mailer from 'common/Mailer';
+import SeriesModel from 'models/Series';
+
+function filterNewReleases(src, dist, props) {
+  /* Series.currentContainsとSeries.recentContainsの差分配列を生成 */
+  return dist.filter((item)=> {
+    let isDiff = true;
+    src.filter((distItem)=> {
+      if (item[props].toString() === distItem[props].toString()) {
+        isDiff = false;
+      }
+      return item[props].toString() !== distItem[props].toString();
+    });
+    return isDiff ? item : null;
+  });
+}
+
+function createSeriesReleases(seriesList) {
+  return seriesList.map((series)=> {
+    return {
+      id: series.id,
+      newReleasesCount: series.currentCount - series.recentCount,
+      newReleases: filterNewReleases(series.recentContains, series.currentContains, '_id'),
+    };
+  });
+}
+
+function createSeriesIDs(seriesList) {
+  return seriesList.map(series => series.id);
+}
+
+function createNewBookIDs(newReleases) {
+  const bookList = [];
+  newReleases.forEach((item)=> {
+    item.newReleases.forEach((book)=> {
+      bookList.push(book._id);
+    });
+  });
+  return bookList;
+}
+
+class PostSeries {
+  constructor() {
+    this.seriesReleases = [];
+    this.seriesIDs = [];
+  }
+
+  getHasNewReleases(done) {
+    const conditions = {
+      hasNewRelease: true,
+    };
+    SeriesModel.find(conditions, (err, series)=> {
+      if (err) {
+        return done(err);
+      }
+      this.seriesReleases = createSeriesReleases(series);
+      this.seriesIDs = createSeriesIDs(series);
+      done(null);
+    });
+  }
+
+  getNewBookIDs(user, done) {
+    const seriesList = user.seriesList;
+    const seriesReleases = this.seriesReleases;
+
+    const notifyList = seriesReleases.filter((item)=> {
+      let notifyItem;
+      seriesList.forEach((series)=> {
+        if (series._id.toString() === item.id.toString()) {
+          notifyItem = item;
+        }
+      });
+      return notifyItem;
+    });
+    done(null, createNewBookIDs(notifyList));
+  }
+
+  sendMail(user, books, done) {
+    const mail = Mailer({ to: user.mail });
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).